string_scrap.js

```diff
@@ -1,31 +1,43 @@
+//1
```

Comment:
Please review string lesson and experiment with different string methods included in the lesson. It is absolutely crucial that you become comfortable with manipulating strings as it is very important for mastery of Javascript. Also read questions more carefully next time as I see instances of codes outputting result that are different from what questions asked for. Come see me or Alejo with questions after reviewing this assignment.